src/main/core/GeoIp/Database/MaxmindGeoIpScriptHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Claroline\CoreBundle\GeoIp\Database;
+
+use Claroline\BundleRecorder\Logger\ConsoleIoLogger;
+use Composer\Script\Event;
+
+class MaxmindGeoIpScriptHandler
+{
+    public static function refreshDatabase(Event $event): void
+    {
+        $logger = new ConsoleIoLogger($event->getIo());
+        $geoIpLicenseKey = getenv('MAXMIND_GEOIP_LICENSE_KEY');
```

Comment:
You should reuse the `platform_options` param you've declared.

FMI : Do we need a different key for each platform ?